eox_core/api/v1/serializers.py

```diff
@@ -171,9 +172,25 @@ class EdxappCourseEnrollmentSerializer(serializers.Serializer):
     username = serializers.CharField(max_length=USERNAME_MAX_LENGTH, default=None, source='user')
     is_active = serializers.BooleanField(default=True)
     mode = serializers.CharField(max_length=100)
-    enrollment_attributes = EdxappEnrollmentAttributeSerializer(many=True, default=[])
+    enrollment_attributes = EdxappEnrollmentAttributeSerializer(many=True, required=False)
```

Comment:
why was the default deleted here?

Reply:
drf-yasg creates an openapi object based on the type of fields of the serializer, because `enrollment_attributes` is a nested serializer and we are using a default it tries to explore the `EdxappEnrollmentAttributeSerializer` fields against an empty list resulting on a traceback error.

Reply:
Ok. If you tested this and it does not affect the expected behavior of the request then lgtm